### ISSUE TITLE
Consistent terminology

### ISSUE THREAD

**Issue by NobodyForNothing** [NobodyForNothing/blood-pressure-monitor-fl#328]
Over time the code used different words for the same thing:
- "analyzer" vs "analyser", 
- "model" vs "repository"
- ...

These should be found and removed once data model migration is complete.

**Comment by derdilla** [comment]
Has been cleaned up where useful. If there are major problems left, consider opening a new issue.
